index.html

```diff
@@ -1382,6 +1375,62 @@ <h2>
             </li>
           </ol>
         </section>
+        <section>
+          <h3>
+            Processing the manifest without a document
+          </h3>
+          <p>
+            The [=processing a manifest=] steps are invoked by [[HTML]]'s
+            processing steps for the [^link^] element, but MAY also be invoked
+            by the user agent to process a manifest without an associated
+            document.
+          </p>
+          <p>
+            In this case, the user agent SHOULD ensure that, at least at some
+            point in the past:
+          </p>
+          <ul>
+            <li>there was at least one page which is [=same origin=] as
+            |document URL:URL| that has a [^link^] element
+            |linkElement:HTMLLinkElement| with a [^link/rel^] of
+            [^link/rel/manifest^] and a [^link/href^] that resolves to
+            |manifest URL:URL|, and that
+            </li>
+            <li>if |manifest URL| is not [=same origin=] as |document URL|, the
+            |bodyBytes:byte sequence| data was fetched with a [=CORS Request=]
+            whose <a data-xref-type="http-header">`Origin`</a> is |document
+            URL|'s [=url/origin=], and whose [=Request/credentials mode=] is
+            set to the [=CORS settings attribute credentials mode=] for
+            |linkElement|'s [^link/crossorigin^] attribute.
+            </li>
+          </ul>
+          <p>
+            (The [[HTML]] steps automatically ensure this.)
```

Comment:
I wonder if you can can say this directly in the above (something like, "In keeping with [[HTML]] behavior..." or something). It's a bit risky in case it gets moved or whatever, but should be ok... it will also make it more clear what "it" is.

Reply:
Done.